addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -286,7 +285,16 @@ const Link = Widget.extend({
     _getData: function () {
         var $url = this.$('input[name="url"]');
         var url = $url.val();
-        var content = this.$('input[name="label"]').val() || url;
+        let content = this.data.content;
+        if (this.linkEl) {
+            content = this.linkEl.textContent;
+        }
+        if (this.needLabel) {
+            content = this.el.querySelector('input[name="label"]').value;
+        }
+        if (!content) {
+            content = url;
+        }
```

Comment:
As I see that the url can still be modified a few lines below, 
we shouldn't rather do it like that in the "return" ?
`content: content || url,`

Reply:
Hello, I'm not sure this is exactly what we want, if there is no label nor text content, and the user inputs `0032...`, the url shown after _correctLink will be: `tel:0032...`, and the same for `mailto:`, even if the user didn't input these prefixes

Reply:
I see that when I create a link from nothing (just clicking on the link button in the toolbar when nothing is selected) and that I type an email adress.. In the Dom, I get only the first letter of the adress, It's looks like a bug with your PR because in runbot it works well..

Reply:
Indeed, by always prioritizing the link text content over the url, I introduced a bug for the newly created links...
We should get the text content over the url only for the edition, if we create a new link, we should take the url as long as the user did not modify the text content himself.
I will add tests so that it will be less likely to break in the future.